.github/workflows/test_helper.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           # TODO: Gradually expand this to cover more products
           sqlfluff lint products/template/
           sqlfluff lint products/cpdb/sql/
-          sqlfluff lint products/knownprojects/
+          sqlfluff lint products/knownprojects/ --templater=jinja
```

Comment:
Would you want to wait till I get dbt templater merged, and you can use that? Or I can try to switch it over in my branch. Either way!

Reply:
I'm down to wait